core/internal/cltest/cltest.go

```diff
@@ -1429,7 +1429,7 @@ func AllExternalInitiators(t testing.TB, store *strpkg.Store) []models.ExternalI
 	t.Helper()
 
 	var all []models.ExternalInitiator
-	err := store.RawDB(func(db *gorm.DB) error {
+	err := store.RawDBWithAdvisoryLock(func(db *gorm.DB) error {
```

Comment:
For getters like these, do we gain anything by keeping any sort of advisory locking around? If these getters are called and then mutations are performed, maybe we could just wrap the whole operation in a `postgres.GormTransaction` (and make the getters accept a `*gorm.DB`)

Reply:
Long term is to just get rid of this!